kinto/core/resource/__init__.py

```diff
@@ -599,7 +613,7 @@ def process_record(self, new, old=None):
 
         return new
 
-    def apply_changes(self, record, changes):
+    def apply_changes(self, record, changes, operations=None):
```

Comment:
This means that existing resources will have to be updated to return the same `requested_changes` that they got in their parameters, but I think that's OK for a little more flexibility.